qemu/tests/cfg/nvdimm.cfg

```diff
@@ -5,14 +5,17 @@
     vm_mem_limit = 30G
     kill_vm_on_error = yes
     login_timeout = 240
-    only x86_64, ppc64le
+    only x86_64, ppc64le, aarch64
     ppc64le:
         required_qemu = [5, )
         only nvdimm_basic nvdimm_negative nvdimm_nvml nvdimm_redis
-        no RHEL.7 RHEL.6 RHEL.5 RHEL.4 RHEL.3
+        no RHEL.7
```

Comment:
Just a bit concern about the commit title `Correcting guest versions on non-x86 platforms`. L20 exclude the RHEL 3,4,5,6, so the guest version is same with before,  The main change is move aarch64 out of unsupported list, how about use this as commit title ?

Reply:
Yep, L12 has some duplicate versions same as L20, so the correction is exactly that.
Will make the commit information clearer.